homeassistant/components/netatmo/sensor.py

```diff
@@ -113,9 +113,9 @@ class NetatmoSensorEntityDescription(SensorEntityDescription, NetatmoRequiredKey
         name="Pressure",
         netatmo_name="pressure",
         entity_registry_enabled_default=True,
-        native_unit_of_measurement=UnitOfPressure.MBAR,
+        native_unit_of_measurement=UnitOfPressure.HPA,
```

Comment:
```suggestion
        native_unit_of_measurement=UnitOfPressure.MBAR,
```

Reply:
Let's revert this change since:
a) Frenck does not like it
b) The unit will be automatically converted to mBar now that the device class is corrected